src/main/java/kr/hellogsm/back_v2/domain/application/dto/request/CreateApplicationReqDto.java

```diff
@@ -0,0 +1,126 @@
+package kr.hellogsm.back_v2.domain.application.dto.request;
+
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import kr.hellogsm.back_v2.domain.application.entity.Application;
+import kr.hellogsm.back_v2.domain.application.entity.admission.AdmissionInfo;
+import kr.hellogsm.back_v2.domain.application.entity.admission.DesiredMajor;
+import kr.hellogsm.back_v2.domain.application.entity.grade.MiddleSchoolGrade;
+import kr.hellogsm.back_v2.domain.application.entity.status.AdmissionStatus;
+import kr.hellogsm.back_v2.domain.application.enums.Gender;
+import kr.hellogsm.back_v2.domain.application.enums.GraduationStatus;
+import kr.hellogsm.back_v2.domain.application.enums.Major;
+import kr.hellogsm.back_v2.global.exception.error.ExpectedException;
+import org.springframework.http.HttpStatus;
+
+import java.time.LocalDate;
+
+/**
+ * 원서 생성을 담당하는 dto 입니다
+ *
+ * @author 변찬우
+ * @since 1.0.0
+ */
+public record CreateApplicationReqDto(
+        @NotBlank
+        String applicantImageUri,
+
+        @NotBlank
+        String applicantName,
+
+        @Enumerated(EnumType.STRING)
+        Gender applicantGender,
+
+        LocalDate applicantBirth,
+
+        @NotBlank
+        String address,
+
+        @NotBlank
+        String detailAddress,
+
+        @NotBlank
+        String graduation,
+
+        @Pattern(regexp = "^(?!\\s*$).+")
```

Comment:
지금 `telephone`(집 전화번호)나 `applicantPhoneNumber`(지원자 휴대폰 번호) 등에서`@Pattern`을 사용해서 검증하고 있는데, 사용하는 정규식을 더 구체적으로 작성해주세요.
현재는 특수문자나 공백정도만 체크하는거 같은데, 더 정확하게 체크했으면 좋겠습니다.

휴대폰 번호의 경우 아래처럼 정규식 사용하지면 될 것 같습니다.
```suggestion
        @Pattern(regexp = "^010(?:\\d{3}|\\d{4})\\d{4}$")
```

Reply:
아래와 같이 정규식을 수정했습니다!
`^0(?:\\d|\\d{2})(?:\\d{3}|\\d{4})\\d{4}$`